apps/sms/views/new_message/style/recipients.css

```diff
@@ -97,7 +97,7 @@ Do not remove.
   width: auto;
 }
 
-.recipient[contenteditable].email {
+.recipient[contenteditable=false].email {
```

Comment:
We moved it to BB at one point because we thought email would use them.
If it's in BB it's safer to do this change only in a separate bug though.